src/helpers/helpersInterfaces.ts

```diff
@@ -0,0 +1,20 @@
+export interface Url {
+  auth: string;
+  locationSearch: string;
+  locationInfo: string;
+  observations: string;
+  current: string;
+  nowCast: string;
+  hourly: string;
+  threeHourly: string;
+  daily: string;
+  airQuality: string;
+}
+
+export interface AxiosOptions {
```

Comment:
you can use AxiosRequestConfig built-in interface to define required options for your requests

Reply:
ok, changed. thank you